### PR TITLE
Feature/tight spot

### PR DESCRIPTION
**Pull request by tovernaar123**
new game called tight spot 
the game start with you being able to fly on your private island . The goal is to make a factorio that can make the objective item which can be seen in the gui.
The player has a currency to buy lang (remove walls) and by items in the market. The currency can be seen in the invtory and in the gui. If you spent more then you have you take a "loan" of 5k coins this debt will be negative points. that summerises the game

### CODE REVIEW

File: modules/mini-games/tightspot.lua [hunk 1]
```diff
@@ -0,0 +1,758 @@
...
+    for i, entity in ipairs(save["entity"]) do
+        local name = entity.name
+        if name ~= "character" then
+            if name == "steel-chest" then
+                chests[#chests+1] = {entity,game.connected_players[1].name}
+            end
+            local position = entity.position
+            local force = entity.force
+            local minbale =  entity.minable
+            local table = {name,position,force,minbale}
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:243]: Good to see local variables but they arent needed here since they are only used once.
```suggestion
            local table = {name,entity.position, entity.force, entity.minable}
```

File: modules/mini-games/tightspot.lua [hunk 2]
```diff
@@ -0,0 +1,758 @@
...
+            if name == "market" then
+                markets[#markets + 1] = position
+            end
+            save["entity"][i] = table
+        else
+            if i == #save["entity"] then
+                save["entity"][i] = nil
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:250]: What is the purpose removing the last entity in the list but only when its a player.

File: modules/mini-games/tightspot.lua [hunk 3]
```diff
@@ -0,0 +1,758 @@
...
+                local position = save["entity"][#save["entity"]].position
+                local force = save["entity"][i].force
+                local minbale =  save["entity"][i].minable
+                local table = {name,position,force,minbale}
+                if name == "market" then
+                    markets[#markets + 1] = position
+                end
+                if name == "steel-chest" then
+                    chests[#chests+1] = {entity,game.connected_players[1].name}
+                end
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:262]: Becuse of the previous if statement you know that this code is only ran for player characters, so what is the point of these checks?

File: modules/mini-games/tightspot.lua [hunk 4]
```diff
@@ -0,0 +1,758 @@
...
+            end
+            save["entity"][i] = table
+        else
+            if i == #save["entity"] then
+                save["entity"][i] = nil
+            else
+                name = save["entity"][#save["entity"]].name
+                local position = save["entity"][#save["entity"]].position
+                local force = save["entity"][i].force
+                local minbale =  save["entity"][i].minable
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:255]: Why are you using the last entity in the list when its a character, but then switching back to the current entity for force and ninable. Also `save["entity"][i]` should just be `entity`

File: modules/mini-games/tightspot.lua [hunk 5]
```diff
@@ -0,0 +1,758 @@
...
+                position = tile.position
+            }
+            save["tiles"][#save["tiles"]+1] = table
+        end
+    end
+
+    save["entity"] = variables["surface"].find_entities_filtered {area = level.area}
+    for i, entity in ipairs(save["entity"]) do
+        local name = entity.name
+        if name ~= "character" then
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:236]: Will there even be a character entity when every player is in a god controler.

File: modules/mini-games/tightspot.lua [hunk 6]
```diff
@@ -0,0 +1,758 @@
...
+
+    local area = variables.level.area
+    area[1][1] = area[1][1]
+    area[2][1] = area[2][1]
+    local left_overs = variables["surface"].find_entities_filtered {area= area}
+    for i, ent in ipairs(left_overs) do
+        if ent.name ~= "market" and ent.name ~= "steel-chest" then
+            ent.destroy()
+        end
+    end
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:351]: You use this cleanup loop a few times, would it be better to have this as a function?

File: modules/mini-games/tightspot.lua [hunk 7]
```diff
@@ -0,0 +1,758 @@
...
+    local players = {}
+    for name,_ in pairs(centers) do
+        players[#players+1] = name
+    end
+    local scores = {}
+    for _,name in ipairs(players) do
+        local player = game.players[name]
+        scores[#scores+1] = {Store.get(balances,player),name}
+    end
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:370]: Why have two loops when it can be done with a single loop?
Also you dont need to get the player object to use Store, when you register the store you are telling it to use the player's name so if you pass the player name then it works just as well as a player object.
```suggestion
    local scores = {}
    for name in pairs(centers) do
        scores[#scores+1] = {Store.get(balances,name), name}
    end
```

File: modules/mini-games/tightspot.lua [hunk 8]
```diff
@@ -0,0 +1,758 @@
...
+local token_for_replace_wall
+local function mined(event)
+    local entity = event.entity
+    if entity.type == "wall" then
+        local position = entity.position
+        local player = game.players[event.player_index]
+        if not variables.walls[player.name][position.x..','..position.y] then
+            local force = entity.force
+            task.set_timeout_in_ticks(1, token_for_replace_wall,{{name = entity.name , position = position , force = force},player.name})
+            player.print("How did you get here.")
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:429]: If this isnt meant to happen you might want to log this event, or at least tell the player to tell an admin.

File: modules/mini-games/tightspot.lua [hunk 9]
```diff
@@ -0,0 +1,758 @@
...
+        end
+    end
+end
+
+local function on_player_left_game(event)
+    local player = game.players[event.player_index]
+
+    local inv = player.get_inventory(defines.inventory.god_main)
+    local all_items = inv.get_contents()
+    game.print(serpent.block(all_items))
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:579]: Left over debug code I assume.

File: modules/mini-games/tightspot.lua [hunk 10]
```diff
@@ -0,0 +1,758 @@
...
+    left_players[player.name] = all_items
+
+    player.set_controller {type = defines.controllers.god}
+    player.create_character()
+    player.teleport({-35,55},"nauvis")
+    --gui
+    local Main_gui = Gui.get_left_element(player, game_gui)
+    local table = Main_gui.container["slider"].table
+    table.visible = false
+    Gui.toggle_left_element(player,  Main_gui, false)
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:589]: Why are you setting controller and gui states for a player who has left the game?

File: modules/mini-games/tightspot.lua [hunk 11]
```diff
@@ -0,0 +1,758 @@
...
+    local items = left_players[player.name]
+    local center = centers[player.name]
+    if items then
+        player.teleport(center,variables.level.surface)
+        if started[1] then
+            player.set_controller {type = defines.controllers.spectator}
+        else
+            player.set_controller {type = defines.controllers.god}
+            local Main_gui = Gui.get_left_element(player, game_gui)
+            Gui.toggle_left_element(player,  Main_gui, true)
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:604]: This will not work, this applies all the times you use it.
Gui.toggle_left_element takes an element define not an element.
This should be `Gui.toggle_left_element(player, game_gui, true)`

File: modules/mini-games/tightspot.lua [hunk 12]
```diff
@@ -0,0 +1,758 @@
...
+
+end
+
+--gui
+local speed_slider
+local  function speed_change(player, _, _)
+    local Main_gui = Gui.get_left_element(player, game_gui)
+    local table = Main_gui.container["slider"].table
+    game.speed = table[speed_slider.name].slider_value
+end
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:623]: The gui event gives you the element, you dont need all this logic to find the slider.
```suggestion
local  function speed_change(player, element, _)
    game.speed = element.slider_value
end
```

File: modules/mini-games/tightspot.lua [hunk 13]
```diff
@@ -0,0 +1,758 @@
...
+Gui.element{
+    type = "slider",
+    minimum_value = 1,
+    maximum_value  = 64,
+    value = 4,
+    value_step = 1,
+}:on_value_changed(speed_change)
+local label_func =
+Gui.element(function(_,parent,name,style,caption)
+    if name ~= nil then
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:637]: You dont need to test for nil. Naming being nil will have the same result regardless of which branch of the if statement is used.

File: modules/mini-games/tightspot.lua [hunk 14]
```diff
@@ -0,0 +1,758 @@
...
+    label_func(scroll_table_labels,"Debt","label","0")
+
+    label_func(scroll_table_labels,nil,"label","Time: ")
+    label_func(scroll_table_labels,"Time","label","10:00")
+
+    label_func(scroll_table2,nil,"label","Type:")
+    label_func(scroll_table2,"Objective","label","Iron gear")
+
+    label_func(scroll_table2,nil,"label","Price: ")
+    label_func(scroll_table2,"prices","label","15")
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:693]: Firstly, I would remove the use of label_func from the two header labels.
This would mean you no longer to to include the style any more.
It would aklso mean you could change the function to make two labels each call:
`label_func(parent, name, caption, default)` -> `label_func(scroll_table2, 'prices', 'Price: ', '15')`

File: modules/mini-games/tightspot.lua [hunk 15]
```diff
@@ -0,0 +1,758 @@
...
+    end
+end
+
+local function change_balance(player,amount)
+    Store.update(balances,player,function()
+        if amount < 0 then
+            local available = player.get_item_count("coin")
+            amount = amount * -1
+            if available < amount then
+                local Debt =  5000 - amount + available
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:116]: cuase you want 5k coins in the inv and the player is still has avaible amount of coins

File: modules/mini-games/tightspot.lua [hunk 16]
```diff
@@ -0,0 +1,758 @@
...
+        local mins = string.format("%02.f", math.floor(seconds/60));
+        local secs = string.format("%02.f", math.floor(seconds  - mins *60));
+        return mins..":"..secs
+    end
+end
+
+local function change_balance(player,amount)
+    Store.update(balances,player,function()
+        if amount < 0 then
+            local available = player.get_item_count("coin")
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:113]: you could dupe coins by putting them in a chest though 😛

File: modules/mini-games/tightspot.lua [hunk 17]
```diff
@@ -0,0 +1,758 @@
...
+
+    local Main_gui = Gui.get_left_element(player, game_gui)
+    Gui.toggle_left_element(player,  Main_gui, true)
+    local table = Main_gui.container["Money"].table
+    table["Balance"].caption = variables.diffuclty
+    table["Time"].caption = SecondsToClock(level.time/60)
+
+    table = Main_gui.container["Objectives"].table
+    table["Objective"].caption = level.objective
+    table["prices"].caption = level.demand.price
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:157]: lol

[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:157]: objectives as its a full table objective shood not be with capital

File: modules/mini-games/tightspot.lua [hunk 18]
```diff
@@ -0,0 +1,758 @@
...
+    local area = {}
+    area[1] = {}
+    area[2] = {}
+    area[1][1] = level.area[1][1]
+    area[1][2] = level.area[1][2]
+    area[2][1] = level.area[2][1]
+    area[2][2] = level.area[2][2]
+    area[1][1] = area[1][1] +  at_player * 500
+    area[2][1] = area[2][1] +  at_player * 500
+    islands[player.name] = area
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:169]: this will increas level_area i tested that but deepcopy seems good

File: modules/mini-games/tightspot.lua [hunk 19]
```diff
@@ -0,0 +1,758 @@
...
+    local left_overs = variables["surface"].find_entities_filtered {area= area}
+    for i, ent in ipairs(left_overs) do
+        if ent.name ~= "market" and ent.name ~= "steel-chest" then
+            ent.destroy()
+        end
+    end
+    local tiles = {}
+    for i,tile in ipairs(save["tiles"]) do
+        tiles[i] = tile
+        tiles[i].position.x = tiles[i].position.x +  at_player * 500
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:179]: at_player is player_offset

File: modules/mini-games/tightspot.lua [hunk 20]
```diff
@@ -0,0 +1,758 @@
...
+    end
+    variables["surface"].set_tiles(tiles)
+    for i, entity in ipairs(save["entity"]) do
+        local name = entity[1]
+        local position = {}
+        position.x = entity[2].x
+        position.y = entity[2].y
+        local force = entity[3]
+        local minable = entity[4]
+        position.x = position.x + at_player * 500
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:189]: see 1 up

File: modules/mini-games/tightspot.lua [hunk 21]
```diff
@@ -0,0 +1,758 @@
...
+        local ent = variables["surface"].create_entity{name = name , position = position , force = force }
+        if name == "steel-chest" then
+            chests[#chests+1] = {ent,player.name}
+        end
+        ent.minable = minable
+    end
+    centers[player.name] = {}
+    centers[player.name].x = level.center.x
+    centers[player.name].y = level.center.y
+    centers[player.name].x =  centers[player.name].x + at_player * 500
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:207]: again we cant cause it will add to level.center which will make it work only once

File: modules/mini-games/tightspot.lua [hunk 22]
```diff
@@ -0,0 +1,758 @@
...
+    variables.walls[player.name] = {}
+    local wal = variables["surface"].find_entities_filtered {name = "stone-wall", area= area}
+    for i,wall in ipairs(wal) do
+        local p = wall.position
+        variables.walls[player.name][p.x..','..p.y] = true
+    end
+    local center = centers[player.name]
+    player.teleport({center.x,center.y},level.surface)
+    game.print("tp")
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:216]: yes

File: modules/mini-games/tightspot.lua [hunk 23]
```diff
@@ -0,0 +1,758 @@
...
+local function level_save()
+    local level = variables.level
+    for x=level.area[1][1],level.area[2][1] do
+        for y = level.area[1][2],level.area[2][2] do
+            local tile = variables["surface"].get_tile(x,y)
+            local table = {
+                name = tile.name,
+                position = tile.position
+            }
+            save["tiles"][#save["tiles"]+1] = table
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:229]: i dont get what you mean here srry

File: modules/mini-games/tightspot.lua [hunk 24]
```diff
@@ -0,0 +1,758 @@
...
+        end
+    end
+
+    save["entity"] = variables["surface"].find_entities_filtered {area = level.area}
+    for i, entity in ipairs(save["entity"]) do
+        local name = entity.name
+        if name ~= "character" then
+            if name == "steel-chest" then
+                chests[#chests+1] = {entity,game.connected_players[1].name}
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:238]: every player is saved but this is the first island which is for the first player

File: modules/mini-games/tightspot.lua (hunk 2 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:250]: cuase this is the save which shood not have charters in it but the table shood still work with ipairs

File: modules/mini-games/tightspot.lua (hunk 3 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:262]: no this is the last element i am chanching the player index to the last so its goan so this is some random entety which is the last in the array

File: modules/mini-games/tightspot.lua (hunk 4 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:255]: second point yes first point no as stated above

File: modules/mini-games/tightspot.lua (hunk 8 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:429]: its an easter egg i gues as it not possible without commands

File: modules/mini-games/tightspot.lua (hunk 9 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:579]: lol yes

File: modules/mini-games/tightspot.lua (hunk 10 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:589]: cuase when he joins back the game might have stoped

File: modules/mini-games/tightspot.lua (hunk 11 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:604]: wierd as it does work 😛

File: modules/mini-games/tightspot.lua (hunk 12 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:623]: lol that much cleaner

File: modules/mini-games/tightspot.lua (hunk 13 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:637]: o okey then it can go

File: modules/mini-games/tightspot.lua (hunk 14 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:693]: ill think about it 👍

File: modules/mini-games/tightspot.lua (hunk 23 above)
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:221]: ok will do

File: modules/mini-games/tightspot.lua (hunk 16 above)
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:119]: ok that seems good

**Comment by tovernaar123** [comment]
that shood be all

File: modules/mini-games/tightspot.lua [hunk 25]
```diff
@@ -0,0 +1,724 @@
...
+        local Main_gui = Gui.get_left_element(player, game_gui)
+        local table = Main_gui.container["slider"].table
+        table.visible = false
+        Gui.toggle_left_element(player,  game_gui, false)
+    end
+    started[1] = false
+
+    local area = variables.level.area
+    area[1][1] = area[1][1]
+    area[2][1] = area[2][1]
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:315]: Does nothing.

File: modules/mini-games/tightspot.lua [hunk 26]
```diff
@@ -0,0 +1,724 @@
...
+    clean_up(area)
+    for i, entity in ipairs(save.entities) do
+        local name = entity[1]
+        local position = entity[2]
+        local force = entity[3]
+        local minable = entity[4]
+        position.x = position.x
+        entity[2].x = position.x
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:323]: Does nothing.

File: modules/mini-games/tightspot.lua [hunk 27]
```diff
@@ -0,0 +1,724 @@
...
+
+local function player_move(event)
+    local player = game.players[event.player_index]
+    if player.surface.name == variables.level.surface then  --check if the player has not been tped away
+        local center = centers[player.name]
+        if center then
+            local pos = player.position
+            local area = islands[player.name]
+            if not insideBox(area,pos) then
+                player.teleport({center.x,center.y},variables.level.surface)
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:531]: Table does not need to be created as a param.
```suggestion
                player.teleport(center, variables.level.surface)
```

File: modules/mini-games/tightspot.lua [hunk 28]
```diff
@@ -0,0 +1,724 @@
...
+    maximum_value  = 64,
+    value = 4,
+    value_step = 1,
+}:on_value_changed(speed_change)
+local label_func =
+Gui.element(function(_,parent,name,caption1,caption2)
+    parent.add {
+        type = "label",
+        caption = caption1,
+        style = "label",
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:602]: `label` is the default style for a label and so does not need to be defined here.

File: modules/mini-games/tightspot.lua [hunk 29]
```diff
@@ -0,0 +1,724 @@
...
+end)
+
+local function clean_up(area)
+    local left_overs = variables["surface"].find_entities_filtered {area= area}
+    for i, ent in ipairs(left_overs) do
+        if ent.name ~= "market" and ent.name ~= "steel-chest" then
+            ent.destroy()
+        end
+    end
+end
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:98]: ah that what invert is for 😛

File: modules/mini-games/tightspot.lua [hunk 30]
```diff
@@ -0,0 +1,724 @@
...
+    if amount < 0 then
+        local available = player.get_item_count("coin")
+        amount = amount * -1
+        if available < amount then
+            local Debt =  5000 - amount
+            player.insert{name = "coin", count = Debt }
+            player.print("Borrowed 5000 coins to pay for the land")
+            local Main_gui = Gui.get_left_element(player, game_gui)
+            local table = Main_gui.container["Money"].table
+            table["Debt"].caption = tostring(tonumber(table["Debt"].caption) + 5000)
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:126]: xD lol yeah ill call it gui_table

File: modules/mini-games/tightspot.lua [hunk 31]
```diff
@@ -0,0 +1,724 @@
...
+    local level = variables.level
+    local tiles = save["tiles"]
+    for x=level.area[1][1],level.area[2][1] do
+        for y = level.area[1][2],level.area[2][2] do
+            local tile = variables["surface"].get_tile(x,y)
+            local table = {
+                name = tile.name,
+                position = tile.position
+            }
+            tiles[#tiles+1] = table
```
[the remark this answers is not in the document]
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:220]: lol if we keep going like this we will get under 500 lines

File: modules/mini-games/tightspot.lua (hunk 25 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:315]: this 100% needed (jk) lol i have no idea why that is their

File: modules/mini-games/tightspot.lua [hunk 32]
```diff
@@ -0,0 +1,743 @@
...
+        tiles[i] = tile
+        tiles[i].position.x = tiles[i].position.x + player_offset
+    end
+    variables["surface"].set_tiles(tiles)
+    for i, entity in ipairs(save.entities) do
+        local name = entity[1]
+        local position = {x = entity[2].x + player_offset, y = entity[2].y}
+        local force = entity[3]
+        local minable = entity[4]
+        position.x = position.x + player_offset
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:190]: This will cause the x position to be offset twice, since line 187 already incudes the player_offset

File: modules/mini-games/tightspot.lua [hunk 33]
```diff
@@ -0,0 +1,743 @@
...
+            }
+        end
+    end
+
+    save.entities = variables["surface"].find_entities_filtered {area = level.area}
+    for i, entity in ipairs(save.entities) do
+        local name = entity.name
+        if name ~= "character" then
+            local table = {name, entity.position, entity.force, entity.minable}
+            save.entities[i] = table
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:233]: This can just be one line since it is so short, same for ln239 and 240

File: modules/mini-games/tightspot.lua [hunk 34]
```diff
@@ -0,0 +1,743 @@
...
+        end
+    )
+    for i, score in ipairs(scores) do
+        local money = score[1]
+        local player_name = score[2]
+        local place = Nth(i)
+        if colors[place] then
+            game.print(colors[place] .. place .. ": " .. player_name .. " with " .. money .. " points.[/color]")
+        else
+            game.print("[color=#808080]" .. place .. ": " .. player_name .. " with " .. money .. " points.[/color]")
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:355]: Might be useful to use string.format. Example `[color=%s]%s: %s with %d points[/color]`

File: modules/mini-games/tightspot.lua [hunk 35]
```diff
@@ -0,0 +1,743 @@
...
+        end
+        for i, player in ipairs(game.connected_players) do
+            local Main_gui = Gui.get_left_element(player, game_gui)
+            local table = Main_gui.container["Money"].table
+            table["Time"].caption = time
+        end
+    end
+end
+
+token_for_replace_wall = Token.register(replace_wall)
```
Comment (Cooldude2606) [inline modules/mini-games/tightspot.lua:511]: Why is this so far away from where it is used?

File: modules/mini-games/tightspot.lua (hunk 34 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:355]: this will become ussless any way (cause of multi server only print in lobby the gold data) but ill change it

File: modules/mini-games/tightspot.lua (hunk 35 above)
Comment (tovernaar123) [reply modules/mini-games/tightspot.lua:511]: idk lol (fixed
